client/sites-dashboard/components/sites-site-url.ts

```diff
@@ -1,14 +1,25 @@
 import styled from '@emotion/styled';
+import { ExternalLink } from '@wordpress/components';
 
-export const SiteUrl = styled.a`
+export const SiteUrl = styled( ExternalLink )`
 	text-overflow: ellipsis;
+	display: inline-flex;
 	overflow: hidden;
-	display: inline-block;
 	font-size: 14px;
+	color: var( --studio-gray-60 ) !important;
+	&:hover {
+		text-decoration: underline;
+	}
+`;
+
+export const Truncated = styled.span`
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+	max-width: 250px;
```

Comment:
Do we need this? The URLs are truncating for me even when disabling this rule.

Reply:
I suppose it's not needed, it truncates when it fills the entire parent width